terraform/stacks/umccr_demo_admin/main.tf

```diff
@@ -0,0 +1,90 @@
+terraform {
+  required_version = ">= 1.5.4"
+
+  backend "s3" {
+    bucket         = "umccr-terraform-states"
+    key            = "umccr_demo_admin/terraform.tfstate"
+    region         = "ap-southeast-2"
+    dynamodb_table = "terraform-state-lock"
+  }
+
+  required_providers {
+    aws = {
+      version = "5.12.0"
+      source  = "hashicorp/aws"
+    }
+  }
+}
+
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+locals {
+  common_tags = {
+    "Environment" : "demo",
+    "Stack" : "umccr_demo_admin",
+    "org.umccr:Product" : "ElsaData"
```

Comment:
The idea was that we accumulate all resources that need `PlatformOwner` permissions in this TF stack. As such I'd keep the Elsa specific tag(s) outside the `common_tags`.

Reply:
ah ok - sure. I thought it was just elsa stuff

Reply:
Should I introduce tags for Elsa specifically on the user and related objects?